### DISCUSSION TITLE
When installing my plugin, I get a NoReverseMatch exception

### DISCUSSION THREAD

**Discussion by Nikita-Fomichev-DiDi** [netbox-community/netbox#11842] [Help Wanted!]
![image](https://user-images.githubusercontent.com/123448748/221585705-5f196b70-e039-494c-b008-7ca240310ed3.png)
Hello, I`m try install my plugin in NetBox. but I see exception "NoReverseMatch"

```
(venv) root@netbox-test:/opt/netbox/netbox#  tail /var/log/syslog
Feb 27 17:08:38 netbox-test gunicorn[42860]:   File "/opt/netbox/venv/lib/python3.9/site-packages/django/template/defaulttags.py", line 238, in render
Feb 27 17:08:38 netbox-test gunicorn[42860]:     nodelist.append(node.render_annotated(context))
Feb 27 17:08:38 netbox-test gunicorn[42860]:   File "/opt/netbox/venv/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
Feb 27 17:08:38 netbox-test gunicorn[42860]:     return self.render(context)
Feb 27 17:08:38 netbox-test gunicorn[42860]:   File "/opt/netbox/venv/lib/python3.9/site-packages/django/template/defaulttags.py", line 472, in render
Feb 27 17:08:38 netbox-test gunicorn[42860]:     url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
Feb 27 17:08:38 netbox-test gunicorn[42860]:   File "/opt/netbox/venv/lib/python3.9/site-packages/django/urls/base.py", line 77, in reverse
Feb 27 17:08:38 netbox-test gunicorn[42860]:     raise NoReverseMatch(
Feb 27 17:08:38 netbox-test gunicorn[42860]: django.urls.exceptions.NoReverseMatch: 'netbox_reports_plugin' is not a registered namespace inside 'plugins'

```

In settings.py in PLUGIN list I of cource added my_plugin.
And I seen, that this [problem is solved](https://github.com/netbox-community/netbox/issues/10982) in release 3.4.0, but but my netbox is version 3.4.2, and I don`t understand what is the reason, maybe in something else?
Thank you in advance!

**Comment by candlerb** [comment]
At https://github.com/netbox-community/netbox/discussions, enter "plugin NoReverseMatch" in the search box, and you will find:
https://github.com/netbox-community/netbox/discussions/11560

It seems the solution is to create an empty filtersets.py file.

**Reply by Nikita-Fomichev-DiDi** [reply]
> NoReverseMatch

Okey, I trying this, but its not helped.
I`m created file filtersets.py in my plugin. And on path netbox/netbox/netbox I see this file filtersets.py is created and is not empty.
